src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -1887,6 +1887,58 @@ public sealed partial class ZoneMembershipCondition : System.Security.ISecurityE
         public System.Security.SecurityElement ToXml(System.Security.Policy.PolicyLevel level) { throw null; }
     }
 }
+namespace System.ServiceProcess.ServiceController
+{
+    public sealed partial class ServiceControllerPermission : System.Security.Permissions.ResourcePermissionBase
+    {
+        public ServiceControllerPermission() { }
+        public ServiceControllerPermission(System.Security.Permissions.PermissionState state) { }
+        public ServiceControllerPermission(System.ServiceProcess.ServiceController.ServiceControllerPermissionAccess permissionAccess, string machineName, string serviceName) { }
+        public ServiceControllerPermission(System.ServiceProcess.ServiceController.ServiceControllerPermissionEntry[] permissionAccessEntries) { }
+        public System.ServiceProcess.ServiceController.ServiceControllerPermissionEntryCollection PermissionEntries { get { throw null; } }
+    }
+    [Flags]
+    public enum ServiceControllerPermissionAccess
+    {
+        None = 0,
+        Browse = 1 << 1,
+        Control = 1 << 2 | Browse,
+    }
+    [Serializable]
```

Comment:
remove [Serializable] from everything in this PR

Reply:
.. because we are very careful about what we put that on in Core. It is no longer sprinkled around.